Sources/Testing/SourceAttribution/Expression.swift

```diff
@@ -140,10 +139,10 @@ public struct Expression: Sendable {
 
     if let value {
       result.runtimeValueDescription = String(describingForTest: value)
-      result.fullyQualifiedTypeNameOfRuntimeValue = _typeName(type(of: value as Any), qualified: true)
+      result.runtimeValueTypeInfo = TypeInfo(describingTypeOf: value)
```

Comment:
This is initializing an instance describing the **type of** `value`, not describing `value` itself. So that's why I thought it best to include the phrasing "type of" in the parameter. There is a separate initializer which accepts `Any.Type`, and that one I do think we could consider naming `init(of:)`.

Reply:
Free function `name(ofTypeOf:)`? 😇

Reply:
I've changed the other init to `init(describing:)` for better parity with this one